src/spec.tsx

```diff
@@ -165,6 +165,12 @@ describe("<KeyHandler />", () => {
     fireEvent.keyDown(document.body, { code: "KeyC" });
     fireEvent.keyDown(document.body, { code: "KeyA" });
     fireEvent.keyDown(document.body, { code: "KeyD" });
+    // Check to confirm that entering a key that is not a part of the melody in the middle of a melody does nothing
+    fireEvent.keyDown(document.body, { code: "KeyA" });
+    fireEvent.keyDown(document.body, { code: "KeyB" });
+    fireEvent.keyDown(document.body, { code: "KeyT" });
+    fireEvent.keyDown(document.body, { code: "KeyC" });
```

Comment:
Do we need to make some assertions here about the `level1` and `level2` values?

Reply:
I guess the final assertion does the trick, but can we just add an assertion that it doesn't change maybe?

Reply:
The final assertions take all of these into account. Is that an ok way to write tests for such checks?

Reply:
Yeah, I think I'm just suggesting that it's easier to extend and understand if each little block of changes that you have commented here also assert the value hasn't changed. Not a blocker though, so feel free to merge what you have here.